presto-main/src/main/java/com/facebook/presto/sql/gen/JoinCompiler.java

```diff
@@ -98,6 +108,12 @@
             .build(CacheLoader.from(key ->
                     internalCompileHashStrategy(key.getTypes(), key.getOutputChannels(), key.getJoinChannels(), key.getSortChannel())));
 
+    private ScalarFunctionImplementation getOperator(Type key)
+    {
+        Signature signature = registry.resolveOperator(OperatorType.IS_DISTINCT_FROM, ImmutableList.of(key, key));
+        return registry.getScalarFunctionImplementation(signature);
```

Comment:
Inline this method.

(In addition, we generally put private methods near the end, not the top. But it doesn't matter here anymore because we're inlining it.)

(This also supersedes @wenleix's comment about the naming of this function now that we're inlining it. But his recommendations are reasonable.)